lib/ruby_lsp/global_state.rb

```diff
@@ -133,6 +137,9 @@ def apply_options(options)
       end
 
       @supports_request_delegation = options.dig(:capabilities, :experimental, :requestDelegation) || false
+      supported_resource_operations = options.dig(:capabilities, :workspace, :workspaceEdit, :resourceOperations)
+      @supported_resource_operations = supported_resource_operations if supported_resource_operations
```

Comment:
Isn't this the same as:

```
@supported_resource_operations ||= options.dig(:capabilities, :workspace, :workspaceEdit, :resourceOperations)
```

Reply:
No because `@supported_resource_operations` defaults to an empty array to avoid nilability with Sorbet.